src/Components/README.md

```diff
@@ -12,7 +12,7 @@ Aspire components are classic .NET NuGet packages which are designed as highly u
 
 - Each component should provide it's own public and `sealed` `Settings` type.
   > [!NOTE]
-  > This type does not use the name `Options` because it is not an `IOptions`. `IOptions` objects can be configured through dependency injection. These settings needs to be read before the DI container is built, so they can't be `IOptions`.
+  > This type does not use the name `Options` because it is not an `IOptions`. `IOptions` objects can be configured through dependency injection. These settings need to be read before the DI container is built, so they can't be `IOptions`.
```

Comment:
Can I suggest to add a link to https://learn.microsoft.com/dotnet/core/extensions/options somehow?